products/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views import AllLook, LookDetail
```

Comment:
```suggestion
from .views import (
    AllLook,
    LookDetail
)
```

두 개 이상의 클래스를 import 할 때는 줄을 분리해주세요